kgraphql/src/test/kotlin/com/apurebase/kgraphql/schema/SchemaPrinterTest.kt

```diff
@@ -112,6 +112,12 @@ class SchemaPrinterTest {
                         }
                     }
                 }
+                unionProperty("nullablePdf") {
+                    returnType = linked
+                    nullable = true
```

Comment:
With the current implementation it is hard (if not impossible) to determine the Kotlin return type. This might also imped fixing it for lists.